### PR TITLE
Periodic update and cleanup

### PR DESCRIPTION
**Pull request by SannitaSSJ**
- **Animals_CIV4GameText.xml**: general cleanup of most of the entries
- **Apocalypse_CIV4GameText.xml**: fixing French translations + adding/fixing Italian translations
- **AutoLog_Civ4GameText.xml**: fixing roughly 30% of Italian translations + other fixes
- **Bonus_CIV4GameText.xml**: rewriting pedia entries for "Nutella", "Tea", "Terra Sigillata", "Tesseract" + adding "Phoenix", "Rainbow Serpent" + other fixes
- **BuildingsCulture_CIV4GameText.xml**: adding draft English pedia entries for "Ashanti", "Athenians", "Blackfoot", "Chippewa", "Croatian", "Danish", "Epirus", "Fijian", "Ghana", "Liangzhu", "Lydian", "Macedonian", "Malaysian", "Mandan", "Massagetae", "Mississippian", "Nama", "Pictish", "Pima", "Roxolani", "Samoan", "Slovakian", "Songhai", "Tahitian", "Timurid", "Venetian", "Wayana", "Yaghan", "Yanomami", "Yemeni", "Zuni" + other fixes
- **Buildings_CIV4GameText.xml**: removing wrongly/not translated tags + fixing entries for "3D Printing Mill", "Advanced Quality Control", "Animation Studio", "National Legal Database", "The Motherland Calls", "Zizkov Television Tower" + removing entry for "Safety (National Legal Database)" + other fixes
- **CIV4GameTextInfos_GreatPeople.xml**: general cleanup of virtually all entries
- **Civics_CIV4GameText.xml**: general cleanup of most of the entries
- **Clockpunk_CIV4GameText.xml**: completely rewrote the whole file
- **Cultural_Heritage__Promotion_CIV4GameText.xml**: adding Italian translations
- **Entertainer_CIV4GameText.xml**: fixing pedia entry for "Bard"
- **Global_CIV4GameText.xml**: general cleanup for most of the entries
- **IdeaClass_CIV4GameText.xml**: adding Italian translations
- **Israel_Civ4GameText.xml**: fixing Italian translations + other fixes
- **NamesCivs_CIV4GameText.xml**: general cleanup for most of the entries
- **P2K_CIV4GameText.xml**: restoring French and Italian translations for "Abandoned Civilization"
- **Pedia_CIV4GameText.xml**: correcting initial quote + other fixes
- **Pepper2000_Building_CIV4GameText.xml**: removing not translated tags + other fixes
- **Promotions_TB_CIV4GameText.xml**: adding several Italian translations
- **Revolution_CIV4GameText.xml**: general cleanup of most of the entries
- **Rhinoceros_CIV4GameText.xml**: several fixes to pedia and strategy entries + other fixes
- **Subdue_Animals_Pedia_CIV4GameText.xml**: rewriting pedia entries for "Dog Kennel" + other fixes
- **Tech_CIV4GameText.xml**: general cleanup of most of the entries
- **Traits_TB_CIV4GameText.xml**: general cleanup of most of the entries
- **Units_CIV4GameText.xml**: general cleanup of most of the entries
- **general cleanups**: fixing/removing wrongly translated/not translated tags + removing duplicates and empty lines + plus fixing and removing a truly horrible quantity of typos

P.S. Shout out to Gryffin and his eagle eyes!

### CODE REVIEW

**Comment by SannitaSSJ** [comment]
I completely reworked the description, in order to better explain what I did in this request. There is one more commit that I plan to do, and then I'll stop for the time being.

File: Assets/XML/Text/Alt_Timelines/Clockpunk/Clockpunk_CIV4GameText.xml [hunk 1]
```diff
@@ -152,18 +152,17 @@
...
 	</TEXT>
 	<TEXT>
 		<Tag>TXT_KEY_TECH_CLOCKPUNK_QUOTE</Tag>
 		<English>"A clockwork heart can't replace the real thing." - Dru Pagliassotti, Clockwork Heart</English>
 		<French>"Un coeur d'horlogerie ne peut pas remplacer la chose réelle." - Dru Pagliassotti, Le Coeur d'Horlogerie</French>
 		<Polish>"Mechaniczne serce może zastąpić rzeczywiste." - Dru Pagliassotti, Clockwork Heart</Polish>
-	</TEXT>
+	</TEXT><!--
```
Comment (raxo2222) [inline Assets/XML/Text/Alt_Timelines/Clockpunk/Clockpunk_CIV4GameText.xml:162]: Commenting text tags will only cause txt key issue, wchich is more spammyu

Comment (SannitaSSJ) [reply Assets/XML/Text/Alt_Timelines/Clockpunk/Clockpunk_CIV4GameText.xml:162]: Ok, I'll take this as an opportunity to rework the file, then. :) The only problem is that it'll take me a while.

Comment (raxo2222) [reply Assets/XML/Text/Alt_Timelines/Clockpunk/Clockpunk_CIV4GameText.xml:162]: > 
> 
> Ok, I'll take this as an opportunity to rework the file, then. :) The only problem is that it'll take me a while.

Its not only this file - search for all >TBD< entries - there were thousand of them or so.

**Comment by Toffer90** [comment]
You got to start making smaller commits to master, 176 text xml files changed, 9000 lines added and 11 000 lines removed in a single commit; it will be a nightmare to merge this with my modmod in one go.

**Comment by SannitaSSJ** [comment]
My apologies :( In my defense, they're all text files, and no python, so it shouldn't affect you much as gameplay

**Pull request merged**